pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.black]
+target-version = ['py36', 'py37', 'py38']
```

Comment:
Also `py39`?

Reply:
See comment in description. I had the same thought but my dreams were crushed by `llvmlite` 😉